Labyrinth/src/LabyrinthOriginal/GameMaster.java

```diff
@@ -17,7 +17,7 @@
 	public static void gameStartUp() {
 		Board b = newBoard(5,5);
 		
-		b.addPlayer(new Player(2,2,Color.green));
+		b.addPlayer(new Player(b.getTiles().get(1),Color.green));
```

Comment:
We should make a method to get a random tile, or use tileAt. As is, this introduces dependencies between the board's representation of tiles and this method - board can't change the result of getTiles without changing this code.